config/test/e2e/persistence/cluster.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: etcd.improbable.io/v1alpha1
+kind: EtcdCluster
+metadata:
+  name: cluster1
```

Comment:
can we name this something that makes it more obvious that its used for persistence tests? to make it easier to debug failing e2e tests

Reply:
I've added some helpers to run each test group in a separate namespace.